consensus/validator.go

```diff
@@ -287,22 +297,12 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	consensus.aggregatedCommitSig = aggSig
 	consensus.commitBitmap = mask
 
-	if recvMsg.BlockNum > consensus.blockNum && recvMsg.BlockNum-consensus.blockNum > consensusBlockNumBuffer {
+	consensus.tryCatchup()
+	if recvMsg.BlockNum > consensus.blockNum {
```

Comment:
do we still need this part when we already have the "spinUpStateSync" above?

Reply:
I think yes. If tryCatchup does not help to get consensus block number to the latest (In most cases, this could be the error of missing state in block verification), committed message is not handled correctly. So I don't want it to interfere with the consensus time out stuff. Thus I return immediately.

Reply:
Approve